generic/container-privileged.yaml

```
@@ -0,0 +1,30 @@
+rules:
+  - id: container-privileged
+    message: Found container command with extended privileges
+    languages: [generic]
+    severity: WARNING
+    metadata:
+      category: security
+      subcategory: [audit]
+      technology: [shell]
+      cwe: "CWE-250: Execution with Unnecessary Privileges"
+      confidence: MEDIUM
+      likelihood: MEDIUM
+      impact: HIGH
+      references:
+        - https://docs.docker.com/engine/reference/commandline/run/
+    pattern-either:
+      - pattern: docker ... --privileged
+      - pattern: docker ... --cap-add=ALL
```

```suggestion
      - pattern: docker ... --cap-add=ALL
      - pattern: docker ... --cap-add=SYS_ADMIN
      - pattern: docker ... --cap-add=SYS_MODULE
```
Could probably add more